--- ./nuttx-env-orig/apps/interpreters/wamr/wamr/product-mini/platforms/nuttx/wamr.mk	2021-03-24 23:13:34.000000000 -0300
+++ ./nuttx-env/apps/interpreters/wamr/wamr/product-mini/platforms/nuttx/wamr.mk	2021-03-31 15:26:38.492959116 -0300
@@ -7,6 +7,8 @@
 
 ifeq ($(CONFIG_ARCH_ARMV7M),y)
 WAMR_BUILD_TARGET := THUMBV7EM
+else ifeq ($(CONFIG_ARCH_ARMV7A),y)
+WAMR_BUILD_TARGET := THUMBV7A
 else ifeq ($(CONFIG_ARCH_ARMV8M),y)
 WAMR_BUILD_TARGET := THUMBV8M
 else ifeq ($(CONFIG_ARCH_X86),y)
